lib/rules/func-call-spacing.js

```diff
@@ -86,9 +86,9 @@ module.exports = {
                 return;
             }
 
-            const hasWhitespace = sourceCode.isSpaceBetweenTokens(prevToken, parenToken);
-            const hasNewline = hasWhitespace &&
-                /\n/.test(text.slice(prevToken.range[1], parenToken.range[0]).replace(/\/\*.*?\*\//g, ""));
+            const textBetweenTokens = text.slice(prevToken.range[1], parenToken.range[0]).replace(/\/\*.*?\*\//g, "");
+            const hasWhitespace = /\s/.test(textBetweenTokens);
+            const hasNewline = hasWhitespace && /[\n\r\u2028\u2029]/.test(textBetweenTokens);
```

Comment:
I am not expert in regex but I think we should use the regex present in [linebreak-style](https://github.com/eslint/eslint/blob/master/lib/rules/linebreak-style.js#L63) rule. 
Even better to store this pattern somewhere central and make both the rule consume that.